seeding-DMLs/SeedingBooks.sql

```diff
@@ -0,0 +1,33 @@
+USE [LibraryManagement]
+GO
+
+-- Insert 1,000 rows of fictional data into the Books table
+DECLARE @Counter INT = 1;
+
+WHILE @Counter <= 1000
+BEGIN
+    INSERT INTO [dbo].[Books] ([BookID], [Title], [Author], [ISBN], [PublishedDate], [Genre], [ShelfLocation], [CurrentStatus])
```

Comment:
Why do we need the `[BookID]` here? It should be an identity column that auto-increments by default.